libs/selectors/src/Autocomplete.tsx

```diff
@@ -986,7 +986,7 @@ function Autocomplete<T extends {}>({
         })}
         {...getComboboxProps({}, { suppressRefError: true })}
       />
-      {allowMultiple && <input type="hidden" name={name} />}
+      {allowMultiple && <input className="absolute w-0 h-0" name={name} />}
```

Comment:
why type hidden doesn't work? it should still be in DOM and visible to JS. This seems to me that you're maybe focusing a wrong element with the helper hook

Reply:
I thought the same, but for some reason when `type="hidden"` it's not found in the DOM. With this workaround it works as expected.